pages/heroku.md

```diff
@@ -131,6 +131,115 @@ Or when using gradle:
+```
+
+If you're using Gradle:
+
+```
+./gradlew -Pprod bootJar jibDockerBuild
+```
+
+You can test it out locally using Docker Compose.
+
+```shell
+docker-compose -f src/main/docker/app.yml up
+```
+
+Once you've confirmed everything works, create a new app on Heroku, and add it as a remote.
+
+```shell
+heroku apps:create
+git remote add docker https://git.heroku.com/<your-new-app>.git
+```
+
+Then run the commands below to deploy your JHipster app as a Docker image. Be sure to replace the `<...>` placeholders with your Heroku app name. If you don't know your app name, run `heroku apps`.
+
+```shell
+heroku container:login
+docker tag space registry.heroku.com/<heroku-app>/web
+docker push registry.heroku.com/<heroku-app>/web
+```
+
+For example:
+
+```shell
+heroku container:login
+docker tag space registry.heroku.com/fast-peak-70014/web
+docker push registry.heroku.com/fast-peak-70014/web
+```
+
+At this point, you can use the PostgreSQL and Okta add-ons you've already configured. Run the following command to get the identifiers of the add-ons from the `heroku` remote that you first deployed to.
+
+```shell
+heroku addons --remote heroku
+```
+
+Then you can attach these instances to your new application.
+
+```shell
+heroku addons:attach <postgresql-addon-name> --remote docker
+heroku addons:attach <okta-addon-name> --remote docker
+```
+
+When you use `jhipster heroku` to deploy your application, it properly configures the database for you. However, when deploying it as a Docker container, none of that happens. Therefore, you need to set a few configuration variables so your Docker container can talk to PostgreSQL. First, run the following command to get the PostgreSQL URL.
+
+```
+heroku config:get DATABASE_URL --remote docker
+```
+
+Then, set the database environment variables to match the keys that are in `application-heroku.yml`:
+
+```shell
+heroku config:set JDBC_DATABASE_URL=<value-after-@-from-last-command> --remote docker
+heroku config:set JDBC_DATABASE_USERNAME=<username-value-from-last-command> --remote docker
+heroku config:set JDBC_DATABASE_PASSWORD=<password-value-from-last-command> --remote docker
+```
+
+Set the max amount of Java memory to use and specify the Spring profiles.
+
+```shell
+heroku config:set JAVA_OPTS=-Xmx256m
+heroku config:set SPRING_PROFILES_ACTIVE=prod,heroku,no-liquibase
+```
+
+Run the command below to open your browser and navigate to your app.
+
+```
+heroku open --remote docker
+```
+
+Copy the URL of your app and log in to your Okta developer account. Go to **Applications** > **Web** > **General** and add the URL to Login and Logout redirect URIs. Make sure the login redirect URI ends with `/login/oauth2/code/oidc`.
+
+NOTE: You will **NOT** be able to login to your JHipster app using the admin account the Okta add-on provisions. If you receive a stack trace when logging in, log out and try again with the account you created when you first deployed to Heroku.
```

Comment:
Maybe something like `to make sure you are not logged in anymore either log out or use a new private window` makes it more clear?